src/xdocs/anttask.xml.vm

```diff
@@ -464,7 +464,33 @@
 
 &lt;/target&gt;
       </source>
+    </section>
 
+    <section name="customJAR">
+      <p><b>Add a custom JAR(like sevntu) to the classpath</b></p>
```

Comment:
Add a custom JAR (with implementation of custom Checks, for example sevntu extension)